package.json

```diff
@@ -24,8 +24,8 @@
     "@angular/platform-browser-dynamic": "^17.1.0",
     "@angular/router": "^17.1.0",
     "@biesbjerg/ngx-translate-extract-marker": "^1.0.0",
-    "@ngx-formly/core": "6.3.2",
-    "@ngx-formly/primeng": "6.3.2",
+    "@ngx-formly/core": "6.3.6",
+    "@ngx-formly/primeng": "6.3.6",
```

Comment:
I will reset the constraint with `^6.3.2`.

Reply:
`6.3.2` to `6.3.5` have bugs though. Better `^6.3.6`?

Reply:
With this solution, he will always take the last one. You can also set the constraint as you mentioned.